src/sentry/integrations/slack/views/link_team.py

```diff
@@ -167,7 +167,7 @@ def handle(self, request: Request, signed_params: str) -> HttpResponseBase:
         analytics.record(
             "integrations.identity_linked",
             provider="slack",
-            actor_id=team.actor_id,
+            actor_id=team.id,
```

Comment:
This is a behavior change but we won't have `actor_id` for long and `team.id` is the next best thing. The `actor_type` attribute should disambiguate what kind of id is being passed.